### ISSUE TITLE
add alt text to splash screen

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/covid-shield-mobile#457]
ENGLISH ALT TEXT: 

Illustration of 8 people enjoying a public space. There’s a person with a walking cane, a person in a wheelchair, a person wearing a hijab, several people of colour, and a representation of gender diversity. 

FRENCH ALT TEXT: 

Illustration de 8 personnes profitant de l’espace public. Il y a une personne avec une canne blanche, une autre en fauteuil roulant, une autre avec un hijab, plusieurs personnes de couleur, et une représentation de la diversité de genre.

**Comment by JuliannaR** [comment]
Temp closing this PR https://github.com/cds-snc/covid-shield-mobile/tree/splash-screen-alt-text 
We will revisit for V2Launch conditions.

**Comment by nickmly** [comment]
According to this: https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/launch-screen

> Avoid including text on your launch screen. Because the content in a launch screen doesn't change, any displayed text won't be localized.

It seems like localized text on the splash screen is not possible on iOS, therefore we cannot do any alt text for it.

**Comment by JuliannaR** [comment]
Lets close this for now and revisit if we need to find a solution re: watermark. https://github.com/cds-snc/covid-shield-mobile/issues/457#issuecomment-655715394
